pom.xml

```diff
@@ -1711,6 +1711,11 @@
                 <artifactId>guice</artifactId>
                 <version>${guice.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.inject.extensions</groupId>
+                <artifactId>guice-assistedinject</artifactId>
+                <version>4.0</version>
```

Comment:
doesn't it work with `${guice.version}` ?

Reply:
it does